wordcloud/wordcloud.py

```diff
@@ -203,6 +208,23 @@ def __init__(self, font_path=None, width=400, height=200, margin=2,
         if font_path is None:
             font_path = FONT_PATH
         self.font_path = font_path
+        ttf = TTFont(
+            font_path, 0, verbose=0, allowVID=0,
+            ignoreDecompileErrors=True,
+            fontNumber=-1
+        )
+        supported_chars = []
+
+        for table in ttf["cmap"].tables:
```

Comment:
Yeah, if someone says :poop: but the font doesn't have it, we skip it so we don't put `[]` in the word cloud